### PR TITLE
Finish point calculation tests cases

### PR DESCRIPTION
**Pull request by selinali2010**

### CODE REVIEW

**Review by RobNadal — APPROVED** [review]

File: gosushi_server/spec/tests/calculatePoints.test.js [hunk 1]
```diff
@@ -530,9 +590,187 @@ describe('calculateSpecialPoints', function () {
     game.players.forEach(p => p.points = 0);
   })
 
+  const removeNigiriPoints = () => {
+    game.players[1].points -= 2;
+    game.players[2].points -= 3;
+    game.players[3].points -= 3;
+  }
+
   it('should give Wasabi points correctly', () => {
```
Comment (RobNadal) [inline gosushi_server/spec/tests/calculatePoints.test.js:599]: This is a pretty useless test name haha - maybe specify (if its short) how wasabi points are calculated? Same for below, if the point calculation is short I would make sense to include it in the test name - ie. it should give _ points for wasabi"

File: gosushi_server/util/calculatePoints.js [hunk 2]
```diff
@@ -14,61 +14,56 @@ const addPlayerPoints = (playerName, players, points) => {
...
-            tripleNigiri = wasabi.length > 0;
+            wasabi.shift();
           }
           break;
         case cardNameEnum.MISO_SOUP:
+          // Find all other players that played a miso
           const repeats = players.filter(
             otherPlyr =>
               otherPlyr.name !== currPlyr.name &&
               otherPlyr.turnCards.find(c => c.name === cardNameEnum.MISO_SOUP)
```
Comment (RobNadal) [inline gosushi_server/util/calculatePoints.js:49]: This is fine but we're going to have to remember to add some 'no duplicate player names' validation on join game

Comment (RobNadal) [reply gosushi_server/util/calculatePoints.js:49]: Oh and also I'm aware this^ is from previous commits its just something to think about in general

File: gosushi_server/spec/tests/calculatePoints.test.js (hunk 1 above)
Comment (selinali2010) [reply gosushi_server/spec/tests/calculatePoints.test.js:599]: The thing is, I'm usually doing a couple vague checks at the same time so idk if we want to specify all of them (each player's hand is a different check). Alternatively, we could break each check up into individual ones? For this one in particular, it's literally not doing any computations - just straight up transferring the data from the card into the player's points. Would it help to briefly explain for each one how the logic works? Like "should give Wasabi points based on card data"?

**Review by hannahli2010 — APPROVED** [review]

**Pull request merged**